src/routes/api/index.js

```diff
@@ -1,31 +1,18 @@
+
 import express from 'express';
 import passport from 'passport';
+import authRouter from './auth';
 
 const usersRouter = express.Router();
 
-usersRouter.post('/users/login', (req, res, next) => {
-  if (!req.body.user.email) {
-    return res.status(422).json({ errors: { email: "can't be blank" } });
-  }
+usersRouter.get('/', (request, response) => response.status(200).send('Welcome to  Shadowcat API'));
 
-  if (!req.body.user.password) {
-    return res.status(422).json({ errors: { password: "can't be blank" } });
-  }
-  passport.authenticate('local', { session: false }, (
-    err,
-    user,
-    info
-  ) => {
-    if (err) {
-      return next(err);
-    }
+usersRouter.use('/auth', authRouter);
```

Comment:
We can prefix this route with the version. `v1`

Reply:
thanks a lot for reviewing my work, have implemented the required changes